match/match.py

```diff
@@ -371,16 +391,77 @@ async def _format_match_message(self, ctx, match_index, user_team_name=None):
         message += "**{0}**\n    versus\n**{1}**\n\n".format(home, away)
         message += "**Lobby Info:**\nName: **{0}**\nPassword: **{1}**\n\n".format(match['roomName'], match['roomPass'])
         message += "**Home Team:**\n{0}\n".format(await self.team_manager.format_roster_info(ctx, home))
-        message += "**Away Team:**\n{0}\n\n".format(await self.team_manager.format_roster_info(ctx, away))
-        message += self._create_additional_info(user_team_name, home, away)
+        message += "**Away Team:**\n{0}\n".format(await self.team_manager.format_roster_info(ctx, away))
+        
+        try:
+            message += self._create_additional_info(user_team_name, home, away, stream_details=match['streamDetails'])
+        except KeyError:
+            message += self._create_additional_info(user_team_name, home, away)
         return message
 
-    def _create_additional_info(self, user_team_name, home, away):
+    async def get_match_from_day_team(self, ctx, match_day, team_name):
+        matches = await self._matches(ctx)
+        # Match format:
+        # match_data = {
+        #     'matchDay': match_day,
+        #     'matchDate': match_date,
+        #     'home': home,
+        #     'away': away,
+        #     'roomName': roomName,
+        #     'roomPass': roomPass
+        # }
+        for match in matches:
+            if match['matchDay'] == match_day:
+                if match['home'].casefold() == team_name.casefold() or match['away'].casefold() == team_name.casefold():
+                    return match
+        return None
+
+    async def set_match_on_stream(self, ctx, match_day, team_name, stream_info):
+        matches = await self._matches(ctx)
+        for match in matches:
+            if not match['matchDay'] == match_day:
+                break 
+            if match['home'] == team_name or match['away'] == team_name:
+                match['streamDetails'] = stream_info
+                await self._save_matches(ctx, matches)
+                return True
+        return False
+
+    async def remove_match_from_stream(self, ctx, match_day, team_name):
+        matches = await self._matches(ctx)
+        for match in matches:
```

Comment:
This loop functionality looks the same as in the get_match_from_day_team method. Is there a reason not to just call that method to get the match and edit it here if found? That way the functionality can be guaranteed to be consistent if it needs to change in the future.